shared-libs/tombstone-utils/src/tombstone-utils.js

```diff
@@ -0,0 +1,96 @@
+var _ = require('underscore');
+
+var TYPES = ['data_record', 'person', 'clinic', 'district_hospital', 'health_center'],
```

Comment:
I'm concerned we'll miss a type if we create a new type later. Would there be much harm in tombstoning everything? Most other docs never get deleted so nothing will change as far as I can tell.

Reply:
I did remove the old types, but added a new one: not creating tombstones for tombstones.